src/components/CsvImport/csv.js

```diff
@@ -42,13 +44,22 @@ export default {
         worker: true,
         comments: false,
         step: undefined,
+        chunk: results => {
+          if (Object.keys(parsedData).length === 0 && parsedData.constructor === Object) {
+            parsedData = results
+          } else {
+            parsedData.data = [...parsedData.data, ...results.data]
+            parsedData.errors = [...parsedData.errors, ...results.errors]
+          }
+        },
+        chunkSize: 1024 * 1024 * 10,
         complete: results => {
```

Comment:
`results` is not used here. It should be function without parameters `complete: () => {`